bin/sdf/src/args.rs

```diff
@@ -202,6 +202,28 @@ pub(crate) struct Args {
     #[arg(long, env = "SI_AUTH_API_URL")]
     pub(crate) auth_api_url: Option<String>,
 
+    /// Override for the openai API key
+    #[arg(long, env = "SI_OPENAI_API_KEY")]
+    // TODO This should be a SensitiveString but if it is, it gets passed as "..."
+    // pub(crate) openai_api_key: Option<SensitiveString>,
+    pub(crate) openai_api_key: Option<String>,
```

Comment:
Yes... good call out. I have a few ideas we can employ here, but this is non-blocking

Reply:
Yeah, I'm quite sure I was just using it wrong and there's a secret incantation I'm missing!